internal/otelcollector/config/metric/input_enabled.go

```diff
@@ -61,3 +61,39 @@ func IsRuntimeVolumeInputEnabled(input telemetryv1alpha1.MetricPipelineInput) bo
 
 	return *input.Runtime.Resources.Volume.Enabled
 }
+
+func IsRuntimeStatefulSetInputEnabled(input telemetryv1alpha1.MetricPipelineInput) bool {
+	// Runtime volume metrics should be disabled by default if any of the fields (Resources, Statefulset or Enabled) is nil
```

Comment:
Please fix this comment. It is statefulset metrics here and not volume metrics.
Similar issue in all of the functions below. Replace `volume metrics` with the correct corresponding resource metrics